docs/README.md

```diff
@@ -231,23 +197,33 @@ source ~/codechecker/venv/bin/activate
 # NOTE: SKIP this line if you want to always specify CodeChecker's full path.
 export PATH=~/codechecker/build/CodeChecker/bin:$PATH
 
-# Path of `scan-build.py` (intercept-build)
-# NOTE: SKIP this line if you don't want to use intercept-build.
-export PATH=~/<user path>/llvm/tools/clang/tools/scan-build-py/bin:$PATH
-
 # Path of the built LLVM/Clang
 # NOTE: SKIP this line if clang is available in your PATH as an installed Linux package.
 export PATH=~/<user path>/build/bin:$PATH
 ```
-
-## Check the test project
+## Analyze the test project
```

Comment:
Because the outer heading is called "analyze..." already, I think this should be changed to "Execute the analyzers" or something like that...